azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+pr:
+  - 5.0
+
+trigger:
+  - 5.0
+
+variables:
+  - group: fabric-variables
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '8.x'
```

Comment:
probably at this point we should move to 10 but that can be done later.

Reply:
I was just leaving it consistent with the existing pipeline, which uses Node 8